exercises/flatten-array/example.js

```diff
@@ -0,0 +1,7 @@
+export default class Flattener {
+  flatten(arr) {
+    return arr
+      .reduce((acc, el) => Array.isArray(el) ? acc.concat(this.flatten(el)) : acc.concat(el), [])
```

Comment:
I think that's a great suggestion @joelwallis !  The more I do this, the more I favor readability.